File: text/ringo/delete/text_delete.js
#! /usr/bin/ringo
// ------------------------------------------------------------------
//
//	text_delete.js
//
//						May/09/2011
//
// ------------------------------------------------------------------
var text_manipulate = require ('/var/www/data_base/common/ringo_common/text_manipulate.js');
// ------------------------------------------------------------------
print	("*** 開始 ***");
var args = require('system').args;
var file_txt = args[1];
var id_in = args[2];
print	(id_in);

var dict_aa = text_manipulate.text_read_proc (file_txt);

delete dict_aa[id_in];

text_manipulate.dict_display_proc (dict_aa);
text_manipulate.text_write_proc (file_txt,dict_aa);

print	("*** 終了 ***");
// ------------------------------------------------------------------

File: text/jruby/delete/text_delete.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	delete/text_delete.rb
#
#				Aug/07/2013
#
# -------------------------------------------------------------
load '/var/www/data_base/common/ruby_common/text_manipulate.rb'
#
# -------------------------------------------------------------
puts "*** 開始 ***"
#
file_in = ARGV[0]
#
key = ARGV[1]
#
puts key
#
dict_aa = text_read_proc(file_in)
#
if (dict_aa.key?(key))
	dict_aa = dict_delete_proc(dict_aa,key)
	dict_display_proc(dict_aa)
	text_write_proc(file_in,dict_aa)
end
#
puts "*** 終了 ***"
#
# -------------------------------------------------------------
